drivers/hmis_external_apis/app/models/hmis_external_apis/external_forms/form_submission.rb

```diff
@@ -106,7 +106,7 @@ def run_form_processor(current_user, project: nil)
         build_enrollment(
           project: project,
           data_source: project.data_source,
-          entry_date: created_at,
+          entry_date: submitted_at, # Enrollment entry date is set to the date of form submission
```

Comment:
I'm not sure about that, was that spec'd somewhere?
It doesn't feel right that if a form was submitted and reviewed 1 week later, than the Entry Date would be backdated. I would assume it's most intuitive for the entry date to be equal to the date when the form was reviewed.

Reply:
Fixed 👍